frontend/src/scenes/insights/InsightTooltip/InsightTooltip.tsx

```diff
@@ -52,7 +52,7 @@ export function InsightTooltip({
             {inspectPersonsLabel && (
                 <div className="inspect-persons-label">
                     <IconHandClick />
-                    Click on data point to view persons
+                    Click on data point to view users or groups
```

Comment:
We should still use persons here given we call them persons in the modal.

cc @clarkus who has been driving us towards more consistent naming through the app.

Reply:
The feature name is persons, so aligning to that makes sense to me. That said, depending on who you ask, you'll hear a different opinion about this. I think we need to reach consensus on nomenclature and then push for an alignment update. I'll file an issue.

Reply:
https://github.com/PostHog/posthog/issues/7643